include/ck/tensor_operation/gpu/device/impl/device_pool3d_fwd_ndhwc_ndhwc.hpp

```diff
@@ -22,6 +24,8 @@ template <typename InDataType,
           typename OutDataType,
           typename IndexDataType, // enable if OutputIndex == true
           typename ComputeDataType,
+          typename InLayout,
+          typename OutLayout,
```

Comment:
Remove InLayout, OutLayout from the implemented device operator but keep them as templates in `DevicePoolFwd`

Reply:
b61339a36b8be395566f11c9e1583315273820c2